force-app/main/default/lwc/assetManagementRelated/assetManagementRelated.html

```diff
@@ -0,0 +1,49 @@
+<!-- sldsValidatorIgnore -->
+<template>
+    <lightning-card title= "Asset Management Subscription">
+        <lightning-datatable key-field="Id" onrowselection = {handleRowSelection}
+                            data = {assetList} 
+                            columns = {columns}>
+        </lightning-datatable>
+        <c-pear-event-manager></c-pear-event-manager>
+        <div slot="actions">
+            <lightning-button-group>
+                <lightning-button variant="neutral" label="Renew Assets" name="Renew" class="slds-var-m-left_x-small" disabled = {activateButtons} onclick = {renewCancelAction}></lightning-button>
```

Comment:
Can you rearrange the properties so they're in the same order on both buttons, please? This will make them easier to compare.

Why does the renew button call `renewCancelAction`, is it renew or cancel, it can't be both. And the cancel button is a popup not an action?